go.mod

```diff
@@ -17,17 +17,9 @@ require (
 	k8s.io/api v0.18.6
 	k8s.io/apiextensions-apiserver v0.18.6
 	k8s.io/apimachinery v0.18.6
-	k8s.io/client-go v8.0.0+incompatible
-	k8s.io/kubernetes v1.14.0
+	k8s.io/client-go v0.18.6
```

Comment:
Please update the dependencies in https://github.com/kubevirt/controller-lifecycle-operator-sdk/blob/master/examples/sample-operator/go.mod as well; older `k8s.io/client-go` is used there.

Reply:
I'm not sure what to update there. The example project uses `github.com/operator-framework/operator-sdk v0.18.0`, and it seems it requires `k8s.io/client-go v0.18.2`.
Updating operator-sdk would need major rewrite.

Reply:
I meant client-go v0.18.2 to v0.18.6 - so it was aligned with the lib version.

Reply:
Ok, Done.